prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -813,14 +902,9 @@ def build_transmission(method="line2sub", **kwargs):
     first_new_id = ac_lines.index.max() + 1
     transformers.index = pd.RangeIndex(first_new_id, first_new_id + len(transformers))
     ac_lines["type"] = "Line"
-    ac_lines["length"] = ac_lines.apply(calculate_branch_mileage, axis=1)
+    ac_lines["length"] = ac_lines.apply(calculate_branch_mileage, axis=1) * 1.6093  # km
```

Comment:
should we add `mile_to_kilometer` to the `const` module?

Reply:
It's already there in `prereise.gather.griddata.transmission.const`. Done.